src/cli/subcommands/snapshot_cmd.rs

```diff
@@ -78,8 +79,9 @@ impl SnapshotCommands {
 
                 let epoch = chain_head.epoch();
 
-                let chain_name = chain_get_name((), &config.client.rpc_token)
+                let chain_name = state_network_name((), &config.client.rpc_token)
```

Comment:
Do we want `state_network_name` rather than `chain_get_name` for compatibility with Lotus?

Reply:
Yes, chain_get_name was a non-standard forest specific API.